runtime/runtime.go

```diff
@@ -3277,18 +3287,21 @@ func aggregateBLSPublicKeys(
 	var err error
 	var key *PublicKey
 	wrapPanic(func() {
-		key, err = runtimeInterface.AggregateBLSPublicKeys(publicKeys)
+		key, err = runtimeInterface.BLSAggregatePublicKeys(publicKeys)
 	})
 
+	// if the crypto layer produces an error, we have invalid input, return nil
 	if err != nil {
-		return nil, err
+		return interpreter.NilValue{}, nil //nolint
```

Comment:
Maybe only nolint the specific lint that is hit (error is ignored?), instead of all linting